dsf-mpi/dsf-mpi-client/pom.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>dsf-mpi-client</artifactId>
+
+	<parent>
+		<artifactId>dsf-mpi-pom</artifactId>
+		<groupId>org.highmed.dsf</groupId>
+		<version>0.2.0-SNAPSHOT</version>
+	</parent>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-context</artifactId>
+		</dependency>
+	</dependencies>
```

Comment:
Can we remove the spring dependency by using the `Function<String, String>` based create method proposed in MasterPatientIndexClientPdqFactory ?